GetMessages/handler.ts

```diff
@@ -62,17 +70,55 @@ type IGetMessagesHandler = (
 export function GetMessagesHandler(
   messageModel: MessageModel
 ): IGetMessagesHandler {
-  return async fiscalCode => {
+  return async (
+    fiscalCode,
+    maybePageSize,
+    maybeEnrichResultData,
+    maybeMaximumId: O.Option<NonEmptyString>,
+    maybeMinimumId: O.Option<NonEmptyString>
+  ) => {
+    const pageSize = pipe(
+      maybePageSize,
+      O.getOrElse(() => defaultPageSize)
```

Comment:
I' m wondering about `defaultPageSize`. If we want to change it we must change this value on io-functions-commons and release a new version. Can we introduce an env variable that could be changed as far as we need ?

Reply:
That could be nice.